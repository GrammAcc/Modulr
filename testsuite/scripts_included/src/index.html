<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Test Page</title>
  </head>
  <body>
    <!-- modulr-component :javascript: -->
    <!-- modulr-component :javascript-module: -->
    <!-- modulr-component :typescript: -->
    <!-- modulr-component :typescript-module: -->
    <main>
      <div>Test Page Body</div>
    </main>
  </body>
</html>
