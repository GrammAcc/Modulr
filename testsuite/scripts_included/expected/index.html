<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Test Page</title>
  </head>
  <body>
    <script src="/components/javascript.js"></script>
    <nav>
      <button>Home</button>
      <button>About</button>
    </nav>
    <script type="module" src="/components/javascript-module.mjs"></script>
    <nav>
      <button>Home</button>
      <button>About</button>
    </nav>
    <script src="/components/typescript.js"></script>
    <nav>
      <button>Home</button>
      <button>About</button>
    </nav>
    <script type="module" src="/components/typescript-module.mjs"></script>
    <nav>
      <button>Home</button>
      <button>About</button>
    </nav>
    <main>
      <div>Test Page Body</div>
    </main>
  </body>
</html>
